cmd/dep/testdata/harness_tests/init/glide/trans-trans-conflict/README.md

```diff
@@ -0,0 +1 @@
+Have two transient dependencies have different versions of the same repo. Resolving should fail.
```

Comment:
This test is asserting an unwanted behavior that will be addressed in #1316. But it's a useful test case for that other story, so let's not remove it. Would you please add a .ignore to the testcase.json, and when we implement the other issue, we'll turn this test on, with the correct expected results (i.e. it should not fail).

Reply:
This test starts to fail since we do not import transient dependencies as constraints of the root repo. This results in incorrect resolving since only constraints that are imported are of direct dependencies of the root. I will add a comment to #1316 to also handle this case explicitly. Shouldn't the resolving for this test fail when the test case is enabled again as part of #1316 ?

Reply:
I believe preferred versions handle this case but currently we have no idea of dealing with conflicting preferred versions.

Reply:
Yeah, I'm a bit fuzzy off the top of my head on the topic of conflicting preferred versions. 😊  But that is a dep-wide question, not just for import, so I'd rather deal with it in that other issue and rope in Sam then.